### PR TITLE
Add twig 2 support & fix tests on Windows

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by WouterJ** [comment]
Everything is now working as it should

**Comment by haroldiedema** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -6,29 +6,30 @@ cache:
...
-      env: DEPS="dev"
+      env: DEPS=dev
 
 before_install:
   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi
   - composer self-update
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony:${SYMFONY_VERSION}; fi
-  - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
+  - if [ "$DEPS" == "dev" ]; then echo "dev!"; fi
+  - perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json;
```
Comment (WouterJ) [inline .travis.yml]: it's still not working correctly, for some reason this line doesn't do the replacing...

Comment (iltar) [inline .travis.yml]:
```
diff --git a/composer.json b/composer.json
index f418c13..17d5b54 100644
--- a/composer.json
+++ b/composer.json
@@ -162,4 +162,4 @@
             "/test"
         ]
     }
-}
+,"minimum-stability":"dev"}
```
When validating `./composer.json is valid, but with a few warnings`, so it seems to work for me. Maybe the bash if is not working as intended?

**Comment by iltar** [comment]
Thanks @WouterJ I will merge it for now and if anything breaks, fix it.

**Pull request merged**